packages/flutter_tools/lib/src/version.dart

```diff
@@ -145,30 +145,21 @@ class FlutterVersion {
   static FlutterVersion get instance => context.putIfAbsent(FlutterVersion, () => new FlutterVersion(const Clock()));
 
   /// Return a short string for the version (`alpha/a76bc8e22b`).
-  static String getVersionString({ bool whitelistBranchName: false }) {
-    String commit = _shortGitRevision(_runSync(<String>['git', 'rev-parse', 'HEAD']));
-    commit = commit.isEmpty ? 'unknown' : commit;
-
-    final String branch = getBranchName(whitelistBranchName: whitelistBranchName);
-
-    return '$branch/$commit';
+  String getVersionString({bool whitelistBranchName: false}) {
```

Comment:
It looks like all callers are passing `whitelistBranchName=true`, so let's nuke the parameter

Reply:
runner.dart does not.